crates/bitwarden-uniffi/android/build-schemas.sh

```diff
@@ -1 +1,7 @@
-cargo run -p uniffi-bindgen generate ../src/sdk.udl --language kotlin --out-dir sdk/src/main/java
+cargo build --package bitwarden-uniffi --release --all-features
+cargo run -p uniffi-bindgen generate \
+  ../../../target/aarch64-linux-android/release/libbitwarden_uniffi.so \
```

Comment:
On the previous line we're compiling for the host target, so I think it would make more sense to use the host build artifact, otherwise we depend on the user having compiled it themselves previously.
```suggestion
  ../../../target/release/libbitwarden_uniffi.so \
```